ivy/functional/backends/jax/experimental/statistical.py

```diff
@@ -197,8 +197,13 @@ def nanmedian(
     overwrite_input: bool = False,
     out: Optional[JaxArray] = None,
 ) -> JaxArray:
+    if overwrite_input:
+        copied_input = input.copy()
+        return jnp.nanmedian(
+            copied_input, axis=axis, keepdims=keepdims, overwrite_input=False, out=None
+        )
     return jnp.nanmedian(
-        input, axis=axis, keepdims=keepdims, overwrite_input=overwrite_input, out=out
+        input, axis=axis, keepdims=keepdims, overwrite_input=False, out=None
```

Comment:
You should not change the parameters here, they should be passed like they are being passed :+1:

Reply:
> You should not change the parameters here, they should be passed like they are being passed :+1: 
> `overwrite_input=overwrite_input, out=out`

Thanks Zaeem. But jax doesn't support overwrite=True and out != None in their function nativly while all other frameworks does.